src/modules/app/reducers/__tests__/reducers.spec.js

```diff
@@ -27,33 +27,33 @@ describe('endpoint reducers', () => {
   describe('finish fetch reducer', () => {
     it('should return the action payload if the old payload was a fetching payload with the same url', () => {
       const oldPayload = new Payload(true, 'https://url.com')
-      const payload = new Payload(false, 'https://url.com', { someType: 'someValue' }, null)
+      const payload = new Payload<CityModel[]>(false, 'https://url.com', [], null)
```

Comment:
Same (and occurrences below).
Or should we allow both/Switch to this?

Reply:
See above.